android/src/main/java/com/rnldk/RnLdkModule.kt

```diff
@@ -78,10 +85,13 @@ class RnLdkModule(private val reactContext: ReactApplicationContext) : ReactCont
 
     // INITIALIZE THE LOGGER #######################################################################
     // What it's used for: LDK logging
-    val logger = Logger.new_impl { arg: String? ->
-      println("ReactNativeLDK: " + arg)
+    val logger = Logger.new_impl { arg: Record ->
+      if (arg._level == org.ldk.enums.Level.LDKLevel_Gossip) return@new_impl;
+      if (arg._level == org.ldk.enums.Level.LDKLevel_Trace) return@new_impl;
+      if (arg._level == org.ldk.enums.Level.LDKLevel_Debug) return@new_impl;
+      println("ReactNativeLDK: " + arg._args)
```

Comment:
Maybe just move this line up two lines so that it runs even for `Trace`, even if the event-sending doesnt?

Reply:
good idea